src/components/Chat.js

```diff
@@ -232,6 +269,15 @@ export default class Chat extends React.Component {
     onBlur: PropTypes.func,
     onSend: PropTypes.func,
     onButtonClick: PropTypes.func,
+    /** Send Text */
+    sendText: PropTypes.string,
+    /** Properties to display svg icon */
```

Comment:
"...as a send icon. "
What happens if both `svgIcon` and `sendText` are specified?
Should probably be called `sendIcon` or `sendSVGIcon`